.github/workflows/benchmark.yml

```
@@ -0,0 +1,36 @@
+
+jobs:
+  benchmark:
+    name: Run SQL Protocol Benchmark
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+
+      - name: Setup Java
+        uses: actions/setup-java@v3
```

actionlint (1.7.4)

the runner of "actions/setup-java@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)